### PR TITLE
Add linkable list projects to index view, sorted by reverse method

### PR DESCRIPTION
**Pull request by middaysan**

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class HomeController < ApplicationController
-  # def index
-  # end
+  def index
+    @projects = Project.last(9).reverse
```
Comment (RGBD) [inline app/controllers/home_controller.rb]: use order and limit instead of last and reverse

File: app/controllers/projects_controller.rb [hunk 2]
```diff
@@ -6,12 +6,16 @@ class ProjectsController < ApplicationController
   load_resource except: %i[show index]
 
   def index
-    @projects = Project.default_scoped
     if params[:query].presence
-      @projects = @projects.search(params[:query])
+      @projects = Project.includes(:user)
+                         .search(params[:query])
+                         .sort_by do |p|
```
Comment (RGBD) [inline app/controllers/projects_controller.rb]: Fix indent, change, use reorder(:created_at)
Also this order should be enforced in else branch without code duplication.

Comment (middaysan) [reply app/controllers/projects_controller.rb]: done1

File: app/controllers/home_controller.rb (hunk 1 above)
Comment (middaysan) [reply app/controllers/home_controller.rb]: done!

**Pull request merged**